test.js

```diff
@@ -1,22 +1,47 @@
-import test from 'ava';
-import fn from './';
+const test = require('ava');
+const tasklist = require('./');
 
-test('main', async t => {
-	const data = await fn();
+const hasDefaultTaskProps = (t, task) => {
+	t.is(typeof task.imageName, 'string');
+	t.is(typeof task.pid, 'number');
+	t.is(typeof task.sessionName, 'string');
+	t.is(typeof task.sessionNumber, 'number');
+	t.is(typeof task.memUsage, 'number');
+};
 
-	t.true(data.length > 0);
-	const d = data[0];
-	t.true(d.imageName.length > 0);
-	t.is(typeof d.pid, 'number');
-	t.is(typeof d.memUsage, 'number');
-});
+const hasNonVerboseTaskProps = (t, task) => {
+	t.is(task.status, undefined);
+	t.is(task.username, undefined);
+	t.is(task.cpuTime, undefined);
+	t.is(task.windowTitle, undefined);
+};
 
-test('filter option', async t => {
-	const data = await fn({filter: ['status ne running']});
+const hasVerboseTaskProps = (t, task) => {
+	t.is(typeof task.status, 'string');
+	t.is(typeof task.username, 'string');
+	t.is(typeof task.cpuTime, 'number');
+	t.is(typeof task.windowTitle, 'string');
+};
 
-	t.true(data.length > 0);
-	const d = data[0];
-	t.true(d.imageName.length > 0);
-	t.is(typeof d.pid, 'number');
-	t.is(typeof d.memUsage, 'number');
-});
+const makeTest = (options, t) => {
```

Comment:
Been using ava since .9, I totally missed that you support test macros now. Thanks for the advice!